oc-chef-pedant/spec/api/controls_spec.rb

```diff
@@ -12,419 +12,438 @@
+          :run_id => "76296628-7de0-4509-adaa-9a525409142d",
+          :start_time => "2014-12-03T00:00:42Z",
+          :end_time => "2014-12-03T00:00:48Z",
+          :control_groups => [
+            {
+              :name => "mysql audit",
+              :status => "failure",
+              :number_success => 1,
+              :number_failed => 1,
+              :id => "59186fdc-2d0a-4653-af83-bbe79302fcaf",
+              :controls => [
+                {
+                  :name => "should be installed",
+                  :status => "success",
+                  :details => "null",
+                  :resource_type => "Package",
+                  :resource_name => "mysql",
+                  :context => [],
+                  :sequence_number => 1
+                },
+                {
+                  :name => "should have mysql service",
+                  :status => "failure",
+                  :details => "expected Service \"mysql\" to be enabled",
+                  :resource_type => "null",
+                  :resource_name => "null",
+                  :context => [],
+                  :sequence_number => 2
+                }
+              ],
+              :cookbook_name => "tyler-mysql",
+              :cookbook_version => "0.1.0",
+              :recipe_name => "default",
+              :line_number => 19
+            }
+          ]
+        }
+      }
 
-      it 'returns a 405 ("Forbidden") for client' do
-        get(api_url("/controls"),
-            client,
-            :payload => input ) do |response|
-              response.should look_like({
-                :status => 405
-              })
-            end
-      end
+      let(:client){ platform.create_client('client.opscode.piab') }
+      let(:clientwithnonode){ platform.create_client('clientwithnonode.opscode.piab') }
+      let(:normaluser){ normal_user }
+      let(:adminuser){ admin_user }
 
-      it 'returns a 405 ("Method Not Allowed") for normal_user' do
-        get(api_url("/controls"),
-            normaluser,
-            :payload => input ) do |response|
-              response.should look_like({
-                :status => 405
-              })
-            end
-      end
+      context 'POST /controls' do
+        after :each do
+          delete(api_url("clients/client.opscode.piab"), platform.superuser)
+          delete("#{platform.server}/clients/client.opscode.piab", platform.superuser)
 
-      it 'returns a 405 ("Method Not Allowed") for admin_user' do
-        get(api_url("/controls"),
-            adminuser,
-            :payload => input ) do |response|
-              response.should look_like({
-                :status => 405
-              })
-            end
-      end
+          delete(api_url("clients/clientwithnonode.opscode.piab"), platform.superuser)
+          delete("#{platform.server}/clients/clientwithnonode.opscode.piab", platform.superuser)
+        end
 
-    end
+        it 'returns a 201 ('Created') for client' do
```

Comment:
@rmoshier - This is the line that's causing your test failures, you have multiple instances of single quotes.